portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,224 @@
+          Google SPS program. Since I was thinking about what I should 
+          put throughout the week, the inputting the text was not too 
+          difficult. Once I arrived at school, I went to retrieve the 
+          homework I needed to grade over the weekend. I then studied 
+          a bit more for my quiz for one of my classes today and organized 
+          one of my organization's drive as a proposal to help with managing 
+          files before I headed to class. I then reviewed a pull request for 
+          my research project and continued to study for my quiz. Afterwards, 
+          I went to class then worked more on the SPS portfolio website while 
+          waiting for my fifth class. Once my fifth class finished, I worked 
+          more on the SPS portfolio website since the cloudshell wasn't 
+          saving the changes I was making in the afternoon. I then went to 
+          one of my club meetings for a major summit event we are planning the
+          first Friday of March of 2020. I then finished most of the SPS website 
+          at this point since I found out that my error was me changing the 
+          wrong files. Before going to bed, I made minor adjustments to my 
+          flutter mobile research project and created logos for one of the 
+          organization I am in. 
+      </p>
+      <h3>2/19/20</h3>
+      <p>
+          Starting my day at 4am, I went to work and had to reinstall
+          all of my software on my new computer with another individual. 
+          I was able to finish a task given to me and submitted a pull 
+          request while also finishing half way of another task. 
+          Afterwards, I went to run errands then returned home at 7pm. 
+          Then, I worked on my research project to make small adjustments 
+          to the design. Since there was not much communication within my 
+          group, I began to look into some of the code written by others 
+          to modify for future reference. Later on, I created pull request 
+          which was accepted. Later on, I made other minor adjustments 
+          before studying for my quiz until 10pm. 
+      </p>
+      <h3>2/18/20</h3>
+      <p>
+          At the start of the day, I started to finalize my features for 
+          flutter mobile research project. I was able to create my nutritonal 
+          calculator before my first class. Afterwards, I worked three hours 
+          on my mobile project finalizing the embedded website and the 
+          additional screen. Then, I went to my classes and began installing 
+          my environment for my software engineering Flask inventory tracking 
+          system project. Exhausted from programming five hours while studying 
+          for classses, I slept early this day in preparation of my work at 4am 
+          the following day. 
+      </p>
       <div id="greeting-container"></div>
+      <p>Click here to get a random message:</p>
```

Comment:
You probably want to switch order of these 2 components so the random message shows up after "Click here to get...."